### PR TITLE
Add logging to long migrations

### PR DESCRIPTION
**Pull request by zeripath**
Also fix v136 to not use models

Closes #11645 

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

**Comment by zeripath** [comment]
Although an enhancement I think this should be backported because some of these migrations take time and people have complained about poor logging in these already.

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
So this is weird I'm getting a log timeout for test ...

**Comment by 6543** [comment]
It will may pass if #11646 was merged?

**Comment by zeripath** [comment]
@6543 the issue was that I had made it so that the migrations never actually finished! Fixed now!

File: models/migrations/v115.go [hunk 1]
```diff
@@ -40,9 +53,9 @@ func renameExistingUserAvatarName(x *xorm.Engine) error {
 			_ = sess.Rollback()
 			break
 		}
+		start += 50
```
Comment (zeripath) [inline models/migrations/v115.go]:
```suggestion
```

File: models/migrations/v115.go [hunk 2]
```diff
@@ -40,9 +53,9 @@ func renameExistingUserAvatarName(x *xorm.Engine) error {
...
 
 		log.Info("select users [%d - %d]", start, start+len(users))
```
Comment (zeripath) [inline models/migrations/v115.go]:
```suggestion
		log.Info("select users [%d - %d]", start, start+len(users))
		start += 50
```

File: integrations/migration-test/migration_test.go [hunk 3]
```diff
@@ -250,7 +250,6 @@ func doMigrationTest(t *testing.T, version string) {
 		return
 	}
```
Comment (zeripath) [inline integrations/migration-test/migration_test.go]:
```suggestion
	setting.NewXORMLogService(false)
```

File: integrations/migration-test/migration_test.go [hunk 4]
```diff
@@ -249,7 +249,6 @@ func doMigrationTest(t *testing.T, version string) {
 	if !restoreOldDB(t, version) {
 		return
 	}
```
Comment (zeripath) [inline integrations/migration-test/migration_test.go]:
```suggestion
	}

```

**Review by CirnoT — APPROVED** [review]

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**